src/SFA.DAS.EmployerAccounts.Web/Views/EmployerAgreement/AboutYourMou.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@using SFA.DAS.Authorization
+@using SFA.DAS.Authorization.Mvc
+@using SFA.DAS.EmployerAccounts.Web.Helpers
+
+@model OrchestratorResponse<EmployerAgreementViewModel>
+
+@{
+    ViewBag.PageID = "mou-about";
+    ViewBag.Title = "About your MoU";
```

Comment:
is `MoU` correct to be shown in the title?

Reply:
I'm pretty sure it is...EoI, MoU...don't you love interchangeable terms.

Reply:
It should probably be document - didn't have a design when this page was done :)